src/main/java/com/study/realworld/global/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.study.realworld.global.exception;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.METHOD_NOT_ALLOWED;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.HttpRequestMethodNotSupportedException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    private final Logger log = LoggerFactory.getLogger(GlobalExceptionHandler.class);
```

Comment:
```suggestion
    private final Logger log = LoggerFactory.getLogger(getClass());
```
인스턴스 멤버로 Logger 💯 
대신 인스턴스 멤버로 사용하니까 네이밍에 자유로운 `getClass()` 제안 드립니다!

Reply:
그냥 명시적이고 싶어서 뒀는데 굳이 그럴 필요는 없는거같네요 🙇‍♂️